src/modules/hip/handlehip.cpp

```diff
@@ -247,6 +247,7 @@ struct HandleImpl
             hipMalloc(&(this->initHandle->mem.mgpu.charArr[i].charmem) , sizeof(Rpp8s) * this->nBatchSize);
 	    }
         hipMalloc(&(this->initHandle->mem.mgpu.rgbArr.rgbmem) , sizeof(RpptRGB) * this->nBatchSize);
+        hipMalloc(&(this->initHandle->mem.mgpu.maskArr.floatmem) , sizeof(Rpp32f) * 8294400);    // 3840 x 2160
```

Comment:
it is better not name the array with .floatmem (is this a union?). Please keep it simple.

Reply:
This one actually was developed previously for rpphandle from batchpd. Just used it here. Shall look into if changes can be made in a subsequent pr.